packages/minifier-js/minifier.js

```diff
@@ -1,35 +1,78 @@
 var terser;
 
-meteorJsMinify = function (source) {
+const getGlobalDefsOptions = ({ arch }) => ({
+  "Meteor.isServer": false,
+  "Meteor.isTest": false,
+  "Meteor.isDevelopment": false,
+  "Meteor.isClient": true,
+  "Meteor.isProduction": true,
+  "Meteor.isCordova": arch === 'web.cordova',
```

Comment:
What about adding:
"Meteor.isModern": arch !== 'web.browser.legacy'

That's assuming cordova is also modern.